src/More/About.spec.tsx

```diff
@@ -1,44 +1,57 @@
 import React from "react"
 import { Linking } from "react-native"
-import { render, fireEvent } from "@testing-library/react-native"
-import {
-  getApplicationName,
-  getVersion,
-  getBuildNumber,
-} from "react-native-device-info"
+import { render, fireEvent, wait, act } from "@testing-library/react-native"
 
 import AboutScreen from "./About"
-
-jest.mock("react-native-device-info", () => {
-  return {
-    getApplicationName: jest.fn(),
-    getBuildNumber: jest.fn(),
-    getVersion: jest.fn(),
-  }
-})
+import * as NativeModule from "../gaen/nativeModule"
 
 describe("About", () => {
-  it("shows the name of the application", () => {
+  it("shows the name of the application", async () => {
+    jest.useFakeTimers()
+
     const applicationName = "application name"
 
-    ;(getApplicationName as jest.Mock).mockReturnValueOnce(applicationName)
+    const applicationNameSpy = jest
+      .spyOn(NativeModule, "getApplicationName")
+      .mockReturnValue(Promise.resolve(applicationName))
 
     const { getByText } = render(<AboutScreen />)
 
-    expect(getByText(applicationName)).toBeDefined()
+    act(() => {
+      jest.advanceTimersByTime(1000)
```

Comment:
this seems a bit odd to need to manually advance the time?  doesnt wait have a 5 second timeout?